src/OVAL/probes/probe/main.c

```diff
@@ -257,6 +258,7 @@ int main(int argc, char *argv[])
 			if (chdir(rootdir) != 0) {
 				fail(errno, "chdir", __LINE__ -1);
 			}
+			dlopen("libgcc_s.so.1", RTLD_NOW | RTLD_GLOBAL);
```

Comment:
That's probably because pthread depends on libgcc_s.so. On other platforms pthread may depend on something else.

I suggest we don't attempt to fix this and tolerate the error.